data/hlint.yaml

```diff
@@ -983,7 +983,7 @@
 # main = let (first, rest) = (take n l, drop n l) in rest -- splitAt n l
 # main = fst (splitAt n l) -- take n l
 # main = snd $ splitAt n l -- drop n l
-# main = map $ \ d -> ([| $d |], [| $d |])
+# {-# LANGUAGE TemplateHaskell #-}; main = map $ \ d -> ([| $d |], [| $d |])
```

Comment:
It's needed by `refactor` otherwise it crashes. For some test cases I created two copies, one with the pragma and the other with `@NoRefactor` as you suggested.